src/components/stats/playerinfo.svelte

```diff
@@ -30,6 +30,9 @@
 		(player?.monthlyPackageRank !== 'NONE' ? player?.monthlyPackageRank : player?.newPackageRank) ??
 		undefined;
 	$: rank = getRankDefaults(rankName as RankName);
+    
+    let dashId = (i: string) => i.substring(0,8)+"-"+i.substring(8,4)+"-"+i.substring(12,4)+"-"+i.substring(16,4)+"-"+i.substring(20);
```

Comment:
Didn't realize SkyCrypt doesn't take undashed profile Ids. I don't think it's worth it to manually add them back. We can just use the cute name like before